Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -1,7 +1,15 @@
 namespace gdjs {
+  declare var jsSHA: any;
+
   const logger = new gdjs.Logger('Leaderboards');
   export namespace evtTools {
     export namespace leaderboards {
+      const computeDigest = (payload: string): string => {
+        const shaObj = new jsSHA('SHA-256', 'TEXT', { encoding: 'UTF8' });
```

Comment:
out of curiosity, how did you pick this library?

Reply:
Library implementing just a single algorithm and lightweight. Something not focused on performance or completness (we don't want multiple megabytes of dependencies) but something that works entirely in JS (or wasm).